bagua/torch_api/bucket.py

```diff
@@ -225,6 +223,9 @@ def append_decentralized_synchronous_op(
         peer_selection_mode: str = "all",
```

Comment:
need to make clear in doc string if we require "ring" for low precision decentralized

Reply:
> need to make clear in doc string if we require "ring" for low precision decentralized

done